jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -25,6 +26,12 @@ local excludedRules = [
       { alert: 'TargetDown' },
     ],
   },
+  {
+    name: 'kubelet.rules',
+    rules: [
+      { alert: 'node_quantile:kubelet_pleg_relist_duration_seconds:histogram_quantile' },
```

Comment:
I'm not seeing this rule being removed from the assets. Probably because there are several occurrences for different `quantile` label values. But at the same time, it's also used by the `KubeletPlegDurationHigh` alert...

Reply:
I mistook a record for alert in the rule definition😅  As it is used by the alert KubeletPlegDurationHigh , it will be dispensed from this exclusion list.